packages/components/src/index.ts

```diff
@@ -66,6 +66,6 @@ export { WooCommerceWooLogo } from './logos/woocommerce-woo-logo';
 export { default as Swipeable } from './swipeable';
 export { default as DotPager } from './dot-pager';
 export { default as EmbedContainer } from './embed-container';
-export { ExperienceControl } from './experience-control';
+export { ExperienceControl, ExperienceType } from './experience-control';
```

Comment:
We generally don't want to expose types directly if they can be extracted from the component types (e.g. `React.ComponentProps< typeof ExperienceControl >[ 'value' ]`).